apps/paf/fields/index.js

```diff
@@ -641,7 +641,7 @@ module.exports = {
   },
   'crime-another-location-phone': {
     className: ['govuk-input', 'govuk-input--width-20'],
-    validate: [{ type: 'maxlength', arguments: 20 }]
+    validate: ['required', 'internationalPhoneNumber', { type: 'maxlength', arguments: 20 }]
```

Comment:
Will need to up the number for all of them - lines 581, 644, 792, 876, 936, 1192 and any others.